.golangci.next.reference.yml

```diff
@@ -70,6 +70,7 @@ output:
   # - `checkstyle`
   # - `code-climate`
   # - `junit-xml`
+  # - `extended-junit-xml`
```

Comment:
maybe `junit-xml-extended`?

just for better ordering

Reply:
I use the same pattern as `colored-line-number`/`line-number`, `colored-tab`/`tab`.